Sources/SwiftSentry/SentrySDK.swift

```diff
@@ -115,6 +115,45 @@ public enum SentrySDK {
         return SentryId(value: id)
     }
 
+#if os(Windows)
+    // Report an exception record to Sentry. This is a Windows specific function as
+    // it relies on the EXCEPTION_POINTERS type to get the crash stack of the exception.
+    //
+    // It differs from captureException by the fact that it captures the stacktrace of the
+    // exception instead of the current stacktrace.
+    public static func captureExceptionRecord(type: String, description: String, exceptionRecord: UnsafeMutablePointer<EXCEPTION_POINTERS>!) -> SentryId {
```

Comment:
nit: Why make `exceptionRecord` optional here? Feels like it should be non-optional. Null checking of that parameter can happen caller-side.

Reply:
Good point. Fixed.